pkg/okteto/context_test.go

```diff
@@ -42,7 +45,8 @@ func Test_K8sContextToOktetoUrl(t *testing.T) {
 	ctx := context.Background()
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if result := K8sContextToOktetoUrl(ctx, tt.in, "namespace"); result != tt.want {
+			fakeK8sProvider := test.NewFakeK8sProvider([]runtime.Object{})
```

Comment:
Now that you are mocking the kubernetes client, you can add a suite case to cover the case in which the URL is extracted from the namespace annotation https://github.com/okteto/okteto/blob/master/pkg/okteto/context.go#L172

Reply:
I think this is the test you are referring:
https://github.com/okteto/okteto/pull/1905/files#diff-d20b02a443589d3c70663887f0ad6516dac079cb9baed19c795faecce59ecbd7R44

Reply:
oh!, yes, that is the test I was meaning, but as the logic belongs to this function, if we are adding a test here, probably makes sense to have that test also here. Anyways, it is fine if that